docs/docs/user-guide/training-data-format.mdx

```diff
@@ -0,0 +1,659 @@
+---
+id: training-data-format
+sidebar_label: Training Data Format
+title: Training Data Format
+description: Description of the YAML format for training data
+---
+
+## Overview
+
+Rasa Open Source uses [YAML](https://yaml.org/spec/1.2/spec.html) as
+a unified and extendable way to manage all training data,
+including NLU data, stories and rules. 
+
+With the YAML format, training data can be split over any number of YAML files,
+and every file can contain any kind of data. 
+The training data parser will read the top level keys in each file to decide
+what kind of data is in a section at training time.
+
+The [domain](../glossary#domain) uses
+the same YAML format as the training data and can also be split across
+multiple files or combined with training data in one file. The domain includes
+the definitions for [responses](../core/domains#responses) and [forms](../core/forms).
+See the [documentation for domains](../core/domains) to learn more.
+
+### High Level Structure
+
+Each file can contain one or more **keys** with corresponding training
+data. One file can contain multiple keys, as long as there is not more
+than one of a certain key in a single file. The available keys are:
+
+- `version`
+- `stories`
+- `nlu`
+- `rules`
+- `e2e_tests`
```

Comment:
let's put these in the order they're explained. I'm wondering if it makes sense to not group stories and rules under `conversation training data`, and rather explain what each is at the beginning of its section? because they're on the same level as `nlu` and there's no reference to "conversation training data" here, maybe it would make sense? not sure though

Reply:
Well lol it looks like we just moved away from that 😅

Reply:
Just switched this to be one heading today @tmbo  suggestion - https://github.com/RasaHQ/rasa/pull/6357/commits/aba2e2e778c0ebc32bccfc5ddd2e89125dc38f4b

It kinda makes sense to group it because NLU is everything for an NLU model, and then conversational is everything for a dialogue management model.

Reply:
Why can't I see your follow up comment from the files changed tab  🤔 anyway that's fine then.